demos/tests/cases.py

```diff
@@ -476,6 +476,18 @@ def single_option_cases(key, *args):
        })
     )),
 
+    PythonDemo(name='face_detection_mtcnn_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'--no_show': None,
+                          '-i': image_net_arg('00000002')}),
+        [
+            TestCase(options={
```

Comment:
This can be merged with previous `TestCase` instance.

Reply:
done